ansible/playbooks/roles/video-control-server/files/apikeys.php

```diff
@@ -0,0 +1,66 @@
+<?php
+$api_keys = array(
+  "ua2220"=>"d85b3bdd3554ba07",
+  "k4601"=>"7d209f41e1797606",
+  "janson"=>"0f3c7d8c7f25e9cf",
+  "k3201"=>"53366cbf08c108f0",
+  "aw1125"=>"0feccb472a1d6ddd",
+  "h1309"=>"e43b0145b63b8d6e",
+  "h1308"=>"52a70875e2c825cb",
+  "aw1120"=>"b8077470bffeb7f8",
+  "aw1121"=>"a9081fb43cd4e27b",
+  "h1301"=>"86ae9beb62e7c7d6",
+  "h1302"=>"fcc51dd19da7e70f",
+  "ub5132"=>"8568c2eb50a55776",
+  "ud2120"=>"1e1c183b361a7bf7",
+  "aw1126"=>"bb20b8c80033c15b",
+  "k3401"=>"e006aba74a51d11a",
+  "k1105"=>"4c5739a15a8d9dc7",
+  "ud2218a"=>"2f797151fe1d8e48",
+  "k4201"=>"aa0b557e7b6fab13",
+  "ub2147"=>"04e570c920836646",
+  "ua2118"=>"26b50a85fd3978c7",
+  "ub4136"=>"91ac7e78f7c74f4f",
+  "k4401"=>"2e69f70b2d25018e",
+  "h2213"=>"862dc666e0201cc4",
+  "ua2114"=>"2d0bce3d5206830e",
+  "h2214"=>"e4dd7f5898f4cbf7",
+  "h2215"=>"4a35ad57643739af",
+  "ud2119"=>"69172ec14cd03dd8",
+  "ub2252a"=>"da0a8d954c83e446",
+  "ub5230"=>"348ed28d013111ec",
+  "ud2208"=>"5fdc1abf550d9fef",
```

Comment:
If these are meant to be secret API keys, they should probably be in the ansible vault instead?